jest.config.js

```diff
@@ -15,6 +16,13 @@ module.exports = {
             },
         },
     },
+    testRunner: "jest-jasmine2",
+    roots: ['<rootDir>', `<rootDir>/${baseUrl}`],
+    modulePaths: [`<rootDir>/${baseUrl}`],
+    setupFiles: ['jest-date-mock'],
```

Comment:
Why is this required? Is this something that could explicitly stated in our jest-test-setup.ts or generally merged with `setupFIlesAfterEnv`?